code/controllers/subsystem/job.dm

```diff
@@ -560,6 +569,19 @@ SUBSYSTEM_DEF(job)
 	else if((job.auto_deadmin_role_flags & DEADMIN_POSITION_SILICON) && ((CONFIG_GET(flag/auto_deadmin_silicons) && !timegate_expired) || (C.prefs?.toggles & DEADMIN_POSITION_SILICON))) //in the event there's ever psuedo-silicon roles added, ie synths.
 		return C.holder.auto_deadmin()
 
+/datum/controller/subsystem/job/proc/setup_assistant_positions()
+	var/datum/job/job = SSjob.GetJob("Assistant")
+	if(!job)
+		CRASH("setup_assistant_positions(): Assistant job is missing")
+
+	var/assistant_ratio = CONFIG_GET(number/assistant_scaling_coeff)
+	if(assistant_ratio > 0)
+		if(job.spawn_positions > 0)
+			var/assistant_positions = max(3, max(job.spawn_positions, round(unassigned.len / assistant_ratio))) //with a ratio of 8, we'd need a pop of more than 24 to have more than 3 assistants (we'll alway have at least 3 of them)
```

Comment:
```suggestion
			var/assistant_positions = max(3, max(job.spawn_positions, round(unassigned.len / assistant_ratio))) //with a ratio of 8, we'd need a pop of more than 24 to have more than 3 assistants (we'll always have at least 3 of them)
```

Reply:
Also--3 assistants at low pop is a CRAZY low number to me. This doesn't bode well about the change for me, considering a significant amount of assistants are just legitimately new players.

Reply:
the number is just so that AT least there is a minimum, but the actual number is defined by job.spawn_positions that is defined by config (so if you set it to 5, you'll always have  a minimum amount of 5 assistants even at lowpop)

also new players are always able to join as assistant, there is code for it that checks if a player has less playing hours than specified (to resolve this i can increase that number, currently it sits at 14, but maybe 20 could be good